src/inquirer/render/console/_checkbox.py

```diff
@@ -96,6 +96,12 @@ def process_input(self, pressed):
         elif pressed == key.RIGHT:
             if self.current not in self.selection:
                 self.selection.append(self.current)
+        elif pressed == key.CTRL_A:
+            self.selection = [i for i in range(len(self.question.choices))]
+        elif pressed == key.CTRL_R:
```

Comment:
why `CTRL+R`? whats the reasoning behind this shortcut?

Reply:
`R` for a reset. Please suggest any other key that makes better sense.

Reply:
no makes sense, I just wasn'T sure what it stands for and wanted to know